metadata-ingestion/src/datahub/ingestion/source/affirm_artifact.py

```diff
@@ -0,0 +1,143 @@
+import sys
```

Comment:
Maybe we can put this file under an `affirm` module to keep it well isolated? `metadata-ingestion/src/datahub/ingestion/source/affirm/artifact.py`

Reply:
done!